src/main/resources/db.scripts.mvp/R__Session_Template_Data.sql

```diff
@@ -1,463 +1,483 @@
+                    ('BLI_G2_C2','C','2','004', NULL),
+                    ('BLI_G2_C2','C','2','005', NULL),
+                    ('BLI_G2_C2','C','2','006', NULL),
+                    ('BLI_G2_C2','C','2','007', NULL),
+                    ('BLI_G2_C2','C','2','008', NULL),
+                    ('BLI_G2_C2','C','2','025', NULL),
+                    ('BLI_G2_C2','C','2','026', NULL),
+                    ('BLI_G2_C2','C','2','027', NULL),
+                    ('BLI_G2_C2','C','2','028', NULL),
+                    ('BLI_G2_C2','C','2','029', NULL),
+                    ('BLI_G2_C2','C','2','030', NULL),
+                    ('BLI_G2_C2','C','2','031', NULL),
+                    ('BLI_G2_C2','C','2','032', NULL),
+                    ('A_WING','A',NULL,NULL, NULL),
+                    ('OTHER_WINGS','B',NULL,NULL, NULL),
+                    ('OTHER_WINGS','C',NULL,NULL, NULL),
+                    ('OTHER_WINGS','D',NULL,NULL, NULL),
+                    ('OTHER_WINGS','E',NULL,NULL, NULL),
+                    ('OTHER_WINGS','F',NULL,NULL, NULL),
+                    ('OTHER_WINGS','SSCU',NULL,NULL, NULL),
+                    ('OTHER_WINGS','HEALTHCARE',NULL,NULL, NULL)
+        ;
+
+        -- update tmp location table with correct group_id.
+        UPDATE tmp_permitted_session_location SET group_id = tmp_session_location_group.id FROM tmp_session_location_group WHERE tmp_permitted_session_location.group_key = tmp_session_location_group.key;
+
+
+        -- insert data into real permitted session location table from temporary one.
+        INSERT INTO permitted_session_location(id,group_id,level_one_code,level_two_code,level_three_code,level_four_code)
+            SELECT id,group_id,level_one_code,level_two_code,level_three_code,level_four_code FROM tmp_permitted_session_location order by id;
+
+        ALTER SEQUENCE permitted_session_location_id_seq RESTART WITH 105;
+
+
+        -- Create link table data
+        INSERT INTO session_to_location_group(session_template_id, group_id)
+        SELECT st.id, g.id FROM tmp_session_template st
+                                           JOIN tmp_session_location_group g ON POSITION(g.key  IN st.locationKeys)<>0 ORDER BY st.id,g.id;
+
+        -- Location group names are only descriptions they need to be updated when the group locations change
+
+        -- Prisoner category
+
+    -- Create temporary group table
+    CREATE TABLE tmp_session_category_group (
+    id                	serial        NOT NULL PRIMARY KEY,
+    prison_code       	VARCHAR(6)    NOT NULL,
+    prison_id         	int,
+    key          		VARCHAR(50)  NOT NULL,
+    name          	    VARCHAR(100)  NOT NULL
+    );
+
+    -- Category group names are only descriptions they need to be updated when the group categories change
+        INSERT INTO tmp_session_category_group (prison_code,key,name)
+        VALUES
+            ('FNI','FNI_CAT_A_HIGH','Category A High Risk'),
```

Comment:
Given VB-1999, would be preferable for this to be "Category A (High Risk) prisoners" so we can just render it directly?

Reply:
okay sure..